.cascade-code/_config/blogs.vm

```diff
@@ -0,0 +1,38 @@
+#*
+ * Chapman.edu/_config/blogs.vm
+ * Blogs settings.
+ *
+ * Usage:
+ * #import( "/_config/blogs.vm" )
+ *
+ * #set ( $blogID = $currentPage.getChild('system-data-structure').getChild('blogName').value )
+ * #set ( $blogParams = $BLOGS_CONFIG[$blogID] )
+ * #set ( $blogTitle = $blogParams[0] )
+ * #set ( $blogCss = $blogParams[1] )
+ *#
+
+## blog id => [ blog title, css theme class ]
+#set ( $BLOGS_CONFIG = {
+  'academics' : [ 'Higher Ed and Technology: Academics at Chapman', '' ],
+  'alumni' : [ 'Chapman Alumni', '' ],
+  'business' : [ 'business', 'Argyros School of Business and Economics', '' ],
+  'education' : [ 'education', 'Attallah College of Educational Studies', '' ],
```

Comment:
Why do some of these keys have 3 values in the array while others have 2? Also, the order seems wrong for some, like `education` and `business` should be swapped with the second element? Based on the comment at the top of the file: 
```
#set ( $blogTitle = $blogParams[0] )
#set ( $blogCss = $blogParams[1] )
```

Reply:
Yes, this was a mistake leading to some of the buggy behavior Mandy noticed. Good eye.